docs/sources/reference/components/loki/loki.source.podlogs.md

```diff
@@ -39,13 +39,18 @@ The component starts a new reader for each of the given `targets` and fans out l
 
 You can use the following arguments with `loki.source.podlogs`:
 
-| Name         | Type                 | Description                               | Default | Required |
-| ------------ | -------------------- | ----------------------------------------- | ------- | -------- |
-| `forward_to` | `list(LogsReceiver)` | List of receivers to send log entries to. |         | yes      |
+| Name            | Type                 | Description                                                                       | Default | Required |
+| ------------    | -------------------- | --------------------------------------------------------------------------------- | ------- | -------- |
+| `forward_to`    | `list(LogsReceiver)` | List of receivers to send log entries to.                                         |         | yes      |
+| `tail_from_end` | `bool`               | Start reading from the end of the log stream for newly discovered Pod containers. | `false` | no       |
 
 `loki.source.podlogs` searches for `PodLogs` resources on Kubernetes.
 Each `PodLogs` resource describes a set of pods to tail logs from.
 
+When `tail_from_end` is `false` (the default), the component reads all available logs from the Kubernetes API for newly discovered Pod containers.
+For long-running Pods, this can result in a large volume of logs being processed, which may be rejected by the downstream Loki instance if they are too old.
+Set `tail_from_end` to `true` to only read new logs from the point of discovery, ignoring the historical log buffer.
```

Comment:
I don't have a perfect way to state this, CC @clayton-cornell for thoughts, but I think it's important to note that in the implementation you've created we do respect the positions file if it exists.

```suggestion
Set `tail_from_end` to `true` to only read new logs from the point of discovery, ignoring the historical log buffer, unless the pod has a saved position in the component's positions file.
```

Reply:
Hey @dehaansa, @clayton-cornell, I have updated the docs while keeping suggested change in mind, please have a look.